doc/widgets/imageembedding.md

```diff
@@ -40,7 +40,17 @@ Image Embedding offers several embedders, each trained for a specific task. Imag
 Embedders
 ---------
 
-The **Inception v3** we are using is available [here](http://download.tensorflow.org/models/image/imagenet/inception-2015-12-05.tgz). The only difference is that we are not using the final predictions, but the penultimate layer of the model, which represents images with vectors. A simple example of how to use the model can be found [here](https://github.com/tensorflow/models/blob/1af55e018eebce03fb61bba9959a04672536107d/tutorials/image/imagenet/classify_image.py). An [article](https://www.nature.com/articles/s41467-019-12397-x) by Godec et al. (2019) explains how the embeddings work and how to use it in Orange.
+**InceptionV3** is Google’s deep neural network for image recognition. It is trained on the ImageNet data set and performs with a 5.6 % top-5 error at the ImageNet dataset. The model we are using is available [here](http://download.tensorflow.org/models/image/imagenet/inception-2015-12-05.tgz). For the embedding, we use the activations of the penultimate layer of the model, which represents images with vectors. 
+
+**SqueezeNet** is a deep model for image recognition that achieves AlexNet-level accuracy on ImageNet with 50x fewer parameters. The model is trained on the ImageNet dataset. We re-implemented the SqueezeNet by using weights from the [author’s pretrained model](https://github.com/DeepScale/SqueezeNet). We use activations from pre-softmax (`flatten10`) layer as an embedding.
+
+**VGG16** and **VGG19** are deep neural networks for image recognition proposed by Visual Geometry Group from the University of Oxford. It is trained on the ImageNet data set and performs with a 7.4% top-5 error at the ImageNet dataset. We use a [community implementation](https://github.com/machrisaa/tensorflow-vgg) of the network in Tensorflow with original weights. As an embedding, we use activations of the penultimate layer - `fc7`.
```

Comment:
> It is trained on the ImageNet data set

They are trained...

Reply:
Second part also redundant as above. I would say it is obvious from the first part.

Reply:
Both fixed